paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentMethodsUI.kt

```diff
@@ -42,10 +44,10 @@ internal fun PaymentMethodsUI(
     paymentMethods: List<SupportedPaymentMethod>,
     selectedIndex: Int,
     isEnabled: Boolean,
-    onItemSelectedListener: (SupportedPaymentMethod) -> Unit
+    onItemSelectedListener: (SupportedPaymentMethod) -> Unit,
+    state: LazyListState = rememberLazyListState()
```

Comment:
Pulling this out so that I can pass in a specific `firstVisibleItemIndex` in tests.

Reply:
Won't this code below already scroll to have the selected item as the first visible?
```
LaunchedEffect(selectedIndex) {
    state.scrollToItem(selectedIndex, 0)
}
```

Reply:
Yep, you’re right. I initially thought that the `LaunchedEffect` didn’t work properly in the test, but turns out it was because I also passed a wrong `selectedIndex` in my test at the same time.